packages/js/components/src/calendar/style.scss

```diff
@@ -1,3 +1,5 @@
+@import "./react-dates.scss";
```

Comment:
how come you copy/pasted the entire css file of `react-dates.scss` in our repo, we should be able to import it from the installed package.
I tried it locally, it does look like our `css-loader` is a bit broken, so you did have to import it like this:
`@import '../node_modules/react-dates/lib/css/_datepicker.css';`

Reply:
Honestly, I copied the approach `@wordpress/components` had used. It never dawned on me to just import from the installed package. I will look into that. Though, I do wonder if there are any differences... perhaps even "just" the RTL handling.

Reply:
Addressed this in 930cb5c